lib/pushmi_pullyu/fedora_object_fetcher.rb

```diff
@@ -0,0 +1,59 @@
+require 'pushmi_pullyu'
```

Comment:
You are requiring pushmi_pullyu on every one of your files in lib. This probably shouldn't be needed?